pkg/config/derive.go

```diff
@@ -139,3 +142,56 @@ func (derived) StatsdArgs(cs *api.OpenShiftManagedCluster) ([]interface{}, error
 		"-SourceRoleInstance", "OSA",
 	}, nil
 }
+
+func (derived) MDSDConfig(cs *api.OpenShiftManagedCluster) ([]byte, error) {
```

Comment:
@jim-minter I will try to incorporate this into main template engine, Or we want to keep XML out of it? Not sure if this is even possible with current state.

Reply:
this gives us circular imports... need to re-arrange stuff :/